src/redux/missions/missions.js

```diff
@@ -13,10 +13,10 @@ const reducer = (state = initialState, action) => {
       return { ...state, data: [...state.data, ...action.payload] };
     case COMPLETE_STATUS:
       return { ...state, status: 'completed' };
-    case JOIN_MISSION: {
+    case TOGGLE_MISSION: {
```

Comment:
nice, I didn't realize to change it for rockets to toggle instead of reserve

Reply:
Thanks!